agents/craftassist/craftassist_agent.py

```diff
@@ -270,11 +273,29 @@ def perceive(self, force=False):
             # perceive from heuristic perception module
             heuristic_perception_output = self.perception_modules["heuristic"].perceive()
             self.memory.update(heuristic_perception_output)
-        # 4. if semantic segmentation model is initialized, call perceive
```

Comment:
we don't want to keep the semantic segmentation route anymore ?

Reply:
I think we should remove all the old model code and interfaces